oat-sa/extension-tao-itemqti
# Fix/aut 1002/outcome declarations negative values

Related Issue: https://oat-sa.atlassian.net/browse/AUT-1002
Outcome declarations: min value can't be set to negative value

**Pre-requisite:** To have an item created.

**SETPS to test:**

- Checkout to branch: fix/AUT-1002/outcome-declarations-negative-values
- Open an item,  click response and add and add an outcome
- edit the outcome and try to put minimum below 0.
- Negative values should be allowed
- Check that max is always equal or higher than min and there is no other strange behavior

**Actual result:** min will not go below 0 with the arrows and will set itself back to 0 if entered manually
**Expected result:**  Min will go below 0 in min input. 
**Testing env:** TBC
